openff/evaluator/datasets/curation/components/thermoml.py

```diff
@@ -59,23 +59,19 @@ class ImportThermoMLData(CurationComponent):
     @classmethod
     def _download_data(cls, schema: ImportThermoMLDataSchema):
 
-        for journal in schema.journal_names:
+        v2020_tarball = "https://data.nist.gov/od/ds/mds2-2422/ThermoML.v2020-09-30.tgz"
```

Comment:
The 2020 tarball is actually a tarball of the journal tarballs IIRC, so might need to then further extract these.

Reply:
I think another (lower priority issue) is that our link will periodically break if they upload new dated tarballs and remove the old ones.  Not sure the best way to avoid that but most likely would involve @mrshirts coordinating with NIST people.